pkg/snet/network.go

```diff
@@ -605,6 +787,22 @@ func (n *Network) Listen(network string, port uint16) (*Listener, error) {
 			return nil, err
 		}
 
+		return makeListener(lis, network), nil
+	case sudpr.Type:
+		sudprC := n.clients.SudprC()
```

Comment:
Can't client be `nil` here, like with stcph above?

Reply:
@Darkren Well spotted, thanks.